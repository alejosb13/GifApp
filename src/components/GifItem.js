import React from 'react'

export default function GifItem({id,title,url}) {
    

    return (
        <div className="card animate__animated animate__fadeInDown">
            <img src={url} alt={title}/>
            <p>{title}</p>
        </div>
    )
}
