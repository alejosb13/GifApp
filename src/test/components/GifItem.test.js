import React, {Component} from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import GifItem from '../../components/GifItem';

it('Renderiza correctamente', () => {
  const wrapper = shallow(
    <GifItem></GifItem>
  );

  expect(toJson(wrapper)).toMatchSnapshot();
});