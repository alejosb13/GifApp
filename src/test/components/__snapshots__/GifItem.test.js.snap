// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renderiza correctamente 1`] = `
<div
  className="card animate__animated animate__fadeInDown"
>
  <img />
  <p />
</div>
`;

exports[`renders correctly 1`] = `
<div
  className="card animate__animated animate__fadeInDown"
>
  <img />
  <p />
</div>
`;
